vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,166 @@
+---
+title: vcpkg_make_configure
+description: Use vcpkg_make_configure to configure a Make-based project.
+ms.date: 01/30/2024
+---
+# vcpkg_make_configure
+
+Configure a Make-based project.
+
+## Usage
+        
+```cmake
+vcpkg_make_configure(
+    [AUTOCONFIG]
+    [COPY_SOURCE]
+    [DISABLE_MSVC_WRAPPERS]
+    [NO_CPPFLAGS]
+    [ADD_BIN_TO_PATH]
+    [NO_DEFAULT_OPTIONS]
+    [NO_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [SOURCE_PATH <source-path>]
+    [OPTIONS <configure-setting>...]
+    [OPTIONS_RELEASE <configure-setting>...]
+    [OPTIONS_DEBUG <configure-setting>...]
+    [PRE_CONFIGURE_CMAKE_COMMANDS <cmake-command>...]
+    [LANGUAGES <language>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### AUTOCONFIG
+
+Runs autoreconf when set.
+
+### COPY_SOURCE
+
+If specified, the source directory will be copied to the build directory before the configuration process begins. Useful for projects that modify the source tree during configuration or build.
+
+### DISABLE_MSVC_WRAPPERS
+
+Disables the use of wrapper scripts for MSVC tools (cl.exe, windres).
+These wrappers translate and reorder command-line flags to accommodate the expected syntax and operational requirements of these tools. When this option is enabled, the `cl.exe` and `windres` tools will directly interpret command-line flags without any modifications.
```

Comment:
```suggestion
By default, `vcpkg_make_configure` supplies wrappers that translate and reorder 'unix / gcc-style' flags into 'msvc-style' flags. This accommodates build systems that assume unix or gcc conventions.

Setting `DISABLE_MSVC_WRAPPERS` removes these wrappers and allows the underlying make build system to supply 'msvc-style' flags directly without translation. When this option is enabled, the `cl` and `windres` tools will directly interpret command-line flags without any modifications.
```

I'm not sure if it should be `cl.exe` and `windres.exe` or `cl` and `windres` but I don't think one should have `.exe` and the other not. I removed `.exe` in my suggestion as I think it's more common to not explicitly say `.exe`

Reply:
Should there be a note about `NO_MSVC_FLAG_ESCAPING` here?